lib/gitlib.sh

```diff
@@ -40,6 +40,7 @@ BRANCH_REGEX="master|release-([0-9]{1,})\.([0-9]{1,})(\.([0-9]{1,}))*$"
 declare -A VER_REGEX=([release]="v(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-alpha|-beta)*\.*(0|[1-9][0-9]*)?"
                       [dotzero]="v(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.0$"
                       [build]="([0-9]{1,})\+([0-9a-f]{5,40})"
+                      [cibuild]="([0-9]{1,})\+*([0-9a-f]{5,40})*"
```

Comment:
also I suspect this entire thing should be optional. right now `git describe --tags` prints out `v1.3.11-beta.0` and `v1.4.6-beta.0`, where the 0 comes from the tag. `git` doesn't add a 0 if there are no additional commits past the tag.